src/components/WorkOrderDetails.js

```diff
@@ -177,10 +179,12 @@ class WorkOrderDetail extends Component {
     const workOrderId = this.props.match.params.workOrderId;
     return (
       <div>
-        <h1>
-          <FontAwesomeIcon icon={faWrench} />{" "}
-          {this.state.titleData[workOrderFields.header]}
-        </h1>
+        <StyledPageTitle>
```

Comment:
I like the styled component approach since it makes changes uniform across. With small changes like this, it might be overkill, though it sets us up for a more maintainable app in the future. 

My go to in this case would have been just be to use [Bootstrap's spacing utility classes](https://getbootstrap.com/docs/4.1/utilities/spacing/) and try adding a CSS class inside the `PageTitle` component of `mb-3` or `mb-4`. Both work, just a matter of choice.

Reply:
Yep, I made the initial assumption that there were a lot more than 4 titles to update. I need to keep my leanings toward styled components in check. 😆